src/pages/console/auth/user-management.mdx

```diff
@@ -7,11 +7,11 @@ In Amplify Studio, you can create and manage users and groups, edit user attribu
 
 To set authorization rules that allow these users and groups to perform create, read, update, or delete operations on your app data, see [Authorization](/console/authz/authorization).
 
-![user management](/images/studio/auth/user-management.png)
+![User management page in Amplify Studio](/images/studio/auth/user-management.png)
 
 ## To create a user
 
-![create user](/images/studio/auth/create-user.png)
+![Process to create a new Amazon Cognito user](/images/studio/auth/create-user.png)
```

Comment:
Maybe we could say this to give it a little more description about what we're showing:
```suggestion
![Screenshot of form to create a new Amazon Cognito user](/images/studio/auth/create-user.png)
```